### ISSUE TITLE
[Philosphy/Design Thinking / Design Services] Bottom cards copy is small.

### ISSUE THREAD

**Issue by mjabbink** [IBM-Design/design-website#316]
Shouldn’t the copy for these be 16px?

<img width="775" alt="screen shot 2019-02-03 at 5 34 08 pm" src="https://user-images.githubusercontent.com/32881239/52183853-22b2ae80-27da-11e9-88f7-ade5a30972f1.png">

**Comment by mjabbink** [comment]
@RyanCaruthers check with  Jeannie on the spacing between header and copy top please.

**Comment by RyanCaruthers** [comment]
Yes @mjabbink  you are correct, type size should be 16. @alisonjoseph see attached image here below for direction: Type size 16, and image includes vertical spacing call outs.

![screen shot 2019-02-04 at 8 56 49 am](https://user-images.githubusercontent.com/46679352/52216074-0f93f300-285b-11e9-972e-448df61962dc.png)
